.github/workflows/linux_manual.yml

```diff
@@ -12,19 +12,27 @@ on:
         default: 'an-awesome-experiment'
         required: true
         type: string
-      server:
+      experiment-experiment-server:
```

Comment:
is this supposed to be twice or should it be:
```suggestion
      experiment-server:
```

Reply:
Yeah, friggen vscode replaced all when I didn't tell it to 😅